tests/src/python/qgis_wrapped_server.py

```diff
@@ -166,6 +166,8 @@ def do_GET(self, post_body=None):
         headers = {}
         for k, v in self.headers.items():
             headers['HTTP_%s' % k.replace(' ', '-').replace('-', '_').replace(' ', '-').upper()] = v
+        if not self.path.startswith('http'):
+            self.path = "%s://%s:%s%s" % ('https' if https else 'http', QGIS_SERVER_HOST, QGIS_SERVER_PORT, self.path)
```

Comment:
How is it related to the image overflow issue?

Reply:
It's not but while I was there...

Reply:
ok :)